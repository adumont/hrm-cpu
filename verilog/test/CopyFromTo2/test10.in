59 0d
20 71
20 01
12 58
32 2b
31 9f
07 3d
36 d3
04 08
05 b9
06 7a
05 8b
41 1b
10 17
04 3d
45 f7
38 72
01 73
09 80
04 ab
22 8c
49 c2
41 86
24 0f
57 bb
13 19
46 dd
41 8d
31 be
59 32
57 eb
08 fb
05 47
17 2a
43 29
38 7f
58 40
28 4c
04 f5
32 1d
32 bc
29 7a
50 f5
23 10
53 16
41 1e
