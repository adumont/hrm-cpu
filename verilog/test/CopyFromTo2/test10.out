0d
71
01
58
2b
9f
3d
d3
08
b9
7a
8b
1b
17
3d
f7
72
73
80
ab
8c
c2
86
0f
bb
19
dd
8d
be
32
eb
fb
47
2a
29
7f
40
4c
f5
1d
